scanpipe/pipes/d2d.py

```diff
@@ -114,6 +125,55 @@ def get_best_path_matches(to_resource, matches):
     return matches
 
 
+def load_ecosystem_config(pipeline, options):
+    """
+    Add ecosystem specific configurations for each ecosystem selected
+    as `options` to the `pipeline`.
+    """
+    configs_by_ecosystem = {
+        ecosystem.ecosystem_option: ecosystem for ecosystem in ECOSYSTEM_CONFIGS
+    }
```

Comment:
This could be replaced by a `def get_ecosystem_config(ecosystem)` imported from the d2d_config module. See the implementation suggestion below.

Reply:
Added, thanks!